<ui version="4.0" >
 <class>VideoWindow</class>
 <widget class="QMainWindow" name="VideoWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MultipleVideoPlayer</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="multiplevideoplayer.qrc" >
    <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>19</y>
     <width>306</width>
     <height>216</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="0" column="0" colspan="2" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <layout class="QHBoxLayout" name="titleHorizontalLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="titleLabel" >
            <property name="text" >
             <string>Title</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" name="artistHorizontalLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Artist:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="artistLabel" >
            <property name="text" >
             <string>Artist</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" >
         <layout class="QHBoxLayout" name="albumHorizontalLayout" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Album:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="albumLabel" >
            <property name="text" >
             <string>Album</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" >
         <layout class="QHBoxLayout" name="yearHorizontalLayout" >
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Year:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="yearLabel" >
            <property name="text" >
             <string>Year</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" >
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget" >
       <layout class="QVBoxLayout" name="videoLayout" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetMaximumSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" >
     <layout class="QHBoxLayout" name="seekerLayout" >
      <item>
       <widget class="QLCDNumber" name="currentTimeLcdNumber" />
      </item>
      <item>
       <widget class="QLCDNumber" name="totalTimeLcdNumber" />
      </item>
     </layout>
    </item>
    <item row="1" column="1" >
     <layout class="QHBoxLayout" name="volumeLayout" >
      <item>
       <widget class="QToolButton" name="volumeButton" >
        <property name="icon" >
         <iconset resource="multiplevideoplayer.qrc" >
          <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionPlayDVD" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuVideo" >
    <property name="title" >
     <string>Video</string>
    </property>
    <addaction name="actionFullscreen" />
   </widget>
   <widget class="QMenu" name="menuAudio" >
    <property name="title" >
     <string>Audio</string>
    </property>
    <widget class="QMenu" name="menuAudioChannels" >
     <property name="title" >
      <string>Audio Streams</string>
     </property>
     <addaction name="actionAudioStream1" />
     <addaction name="actionAudioStream2" />
     <addaction name="actionAudioStream3" />
     <addaction name="actionAudioStream4" />
     <addaction name="actionAudioStream5" />
    </widget>
    <addaction name="menuAudioChannels" />
   </widget>
   <widget class="QMenu" name="menuSubtitle" >
    <property name="title" >
     <string>Subtitle</string>
    </property>
    <widget class="QMenu" name="menuSubtitle_Streams" >
     <property name="title" >
      <string>Subtitle Streams</string>
     </property>
     <addaction name="actionSubtitleStream1" />
     <addaction name="actionSubtitleStream2" />
     <addaction name="actionSubtitleStream3" />
     <addaction name="actionSubtitleStream4" />
     <addaction name="actionSubtitleStream5" />
    </widget>
    <addaction name="menuSubtitle_Streams" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuVideo" />
   <addaction name="menuAudio" />
   <addaction name="menuSubtitle" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>269</y>
     <width>306</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>235</y>
     <width>306</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay" />
   <addaction name="actionPause" />
   <addaction name="actionStop" />
   <addaction name="separator" />
   <addaction name="actionFullscreen" />
  </widget>
  <action name="actionAddFiles" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text" >
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/qt.png</normaloff>:/images/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlay" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/fullscreen.png</normaloff>:/images/fullscreen.png</iconset>
   </property>
   <property name="text" >
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionPlayDVD" >
   <property name="icon" >
    <iconset resource="multiplevideoplayer.qrc" >
     <normaloff>:/images/dvd.png</normaloff>:/images/dvd.png</iconset>
   </property>
   <property name="text" >
    <string>Play DVD</string>
   </property>
  </action>
  <action name="actionAudioStream1" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>1</string>
   </property>
  </action>
  <action name="actionAudioStream2" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>2</string>
   </property>
  </action>
  <action name="actionAudioStream3" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>3</string>
   </property>
  </action>
  <action name="actionAudioStream4" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>4</string>
   </property>
  </action>
  <action name="actionAudioStream5" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>5</string>
   </property>
  </action>
  <action name="actionSubtitleStream1" >
   <property name="text" >
    <string>1</string>
   </property>
  </action>
  <action name="actionSubtitleStream2" >
   <property name="text" >
    <string>2</string>
   </property>
  </action>
  <action name="actionSubtitleStream3" >
   <property name="text" >
    <string>3</string>
   </property>
  </action>
  <action name="actionSubtitleStream4" >
   <property name="text" >
    <string>4</string>
   </property>
  </action>
  <action name="actionSubtitleStream5" >
   <property name="text" >
    <string>5</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="multiplevideoplayer.qrc" />
 </resources>
 <connections/>
</ui>
